### PR TITLE
FOUR-19753 : The task comments is not showing the buttons to register the comments

### PR DESCRIPTION
**Pull request by henryjonathanquispe**
## Issue & Reproduction Steps
1. Open a task ->  /task
2. Go to comments tab
3. Tried to add a comments

## Solution
- Change the css styles in the view

## How to Test
1. Open a task
2. Verify the tab comments and tried to add a comment

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-19753

## Code Review Checklist
- [ ] I have pulled this code locally and tested it on my instance, along with any associated packages.
- [ ] This code adheres to [ProcessMaker Coding Guidelines](https://github.com/ProcessMaker/processmaker/wiki/Coding-Guidelines).
- [ ] This code includes a unit test or an E2E test that tests its functionality, or is covered by an existing test.
- [ ] This solution fixes the bug reported in the original ticket.
- [ ] This solution does not alter the expected output of a component in a way that would break existing Processes.
- [ ] This solution does not implement any breaking changes that would invalidate documentation or cause existing Processes to fail.
- [ ] This solution has been tested with enterprise packages that rely on its functionality and does not introduce bugs in those packages.
- [ ] This code does not duplicate functionality that already exists in the framework or in ProcessMaker.
- [ ] This ticket conforms to the PRD associated with this part of ProcessMaker.

ci:deploy

### CODE REVIEW

File: resources/views/tasks/edit.blade.php [hunk 1]
```diff
@@ -27,17 +27,17 @@ function() use ($task) {
       ], 'attributes' => 'v-cloak'])
 @endsection
 @section('content')
-  <div id="task" v-cloak>
+  <div id="task" v-cloak class="tw-flex tw-w-full tw-grow">
```
Comment (fagubla) [inline resources/views/tasks/edit.blade.php:30]: this must be in multiple lines

File: resources/views/tasks/edit.blade.php [hunk 2]
```diff
@@ -27,17 +27,17 @@ function() use ($task) {
...
     <div class="menu-mask" :class="{ 'menu-open': showMenu }"></div>
-    <div class="info-main" :class="{ 'menu-open': showMenu }">
-      <div v-cloak class="container-fluid px-3">
-          <div class="d-flex flex-column flex-md-row container-height">
-              <div class="flex-grow-1">
+    <div class="info-main" :class="['tw-w-full tw-flex tw-grow',{ 'menu-open': showMenu }]">
+      <div v-cloak class="tw-flex tw-w-full tw-grow">
```
Comment (fagubla) [inline resources/views/tasks/edit.blade.php:33]: multiple lines

File: resources/views/tasks/edit.blade.php
Comment (fagubla) [inline resources/views/tasks/edit.blade.php:1]: how this is working with intercom?

File: resources/jscomposition/base/form/Dropdown.vue [hunk 3]
```diff
@@ -86,7 +86,7 @@ export default defineComponent({
       default: () => [],
     },
     value: {
-      type: Object,
+      type: [Object, null],
```
Comment (fagubla) [inline resources/jscomposition/base/form/Dropdown.vue:89]: null?
is not better use required as false

File: resources/jscomposition/cases/casesMain/utils/filters.js [hunk 4]
```diff
@@ -12,6 +12,11 @@ export const formatFilters = (filters) => {
       value = e.value.map((o) => o.value);
     }
 
+    // Case status is a dropdown filter, using for the API
+    if (e.field === "case_status") {
```
Comment (fagubla) [inline resources/jscomposition/cases/casesMain/utils/filters.js:16]: which meas 'e'

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-5107741f4d.engk8s.processmaker.net

File: resources/views/tasks/edit.blade.php
Comment (fagubla) [inline resources/views/tasks/edit.blade.php:1]: use a correct code style

**Review by fagubla — CHANGES_REQUESTED** [review]

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-5107741f4d.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-5107741f4d.engk8s.processmaker.net

**Comment by tokensPM** [comment]
QA server K8S was successfully deployed https://ci-5107741f4d.engk8s.processmaker.net

**Review by fagubla — APPROVED** [review]

**Pull request merged**